src/tasks/add-tasks.js

```diff
@@ -0,0 +1,38 @@
+import { addFilter } from '@wordpress/hooks';
+
+const TaskComp = () => {
```

Comment:
what is a `TaskComp`? Could this be inlined in `generateTasks`?

Reply:
I should rename this, it is the container that is shown when clicking on a task.